src/QueryM.purs

```diff
@@ -226,11 +231,19 @@ traceQueryConfig = do
     , usedTxOuts
     , networkId: TestnetId
     , logLevel
+    , pparams
     }
   where
   logLevel :: LogLevel
   logLevel = Trace
 
+getProtocolParametersAff
+  :: OgmiosWebSocket -> LogLevel -> Aff Ogmios.ProtocolParameters
```

Comment:
minor nitpick would be to switch the order of `OgmiosWebSocket` and `LogLevel` so it's inline with `mkDatumCacheWebSocketAff` and `mkOgmiosWebSocketAff`